### PR TITLE
Update installation wording on par with Xcode 14

### PR DESCRIPTION
**Pull request by iiqrah**
While forking/cloning and getting the app to run locally, I found the steps mentioned in the Installation section to be outdated for Xcode 14. 

I have gone ahead and made the changes on par with Xcode 14. 

Let me know if you'd like me to modify the changes in any way. 

Thanks! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,13 @@ Please check the [Sample App](https://github.com/metaplex-foundation/metaplex-io
 
 ## Installation
 
-From Xcode 11, you can use [Swift Package Manager](https://swift.org/package-manager/) to add Solana.swift to your project.
+From Xcode 14, you can use [Swift Package Manager](https://swift.org/package-manager/) to add Solana.swift to your project.
```
Comment (mike-metaplex) [inline README.md:11]: So this is actually an error from the previous documentation. While technically yes, we are adding `Solana.swift` the our project, we are adding more than that. So let's replace `Solana.swift` to `Metaplex-iOS`

**Review by mike-metaplex — COMMENTED** [review]
Great work! Thanks so much for this PR and updating the docs! Just the one small change because of an error in our old documentation.

File: README.md (hunk 1 above)
Comment (iiqrah) [reply README.md]: Got it, thanks for explaining that! I've made the necessary change :))

**Comment by iiqrah** [comment]
Heya @mike-metaplex, just wondering how to get this PR merged 😅

**Comment by mike-metaplex** [comment]
> Heya @mike-metaplex, just wondering how to get this PR merged 😅

It looks like I'm not able to re-run the build checks. Also I tried to pull your branch to rebase, but I'm not able to. Could you push your branch again please?

**Comment by iiqrah** [comment]
I am not too sure if I pushed it correctly as intended but hopefully the build check will pass now? Or we can close this PR and I can do a fresh set-up and make the README changes. I noticed that for some reason both my GitHub accounts are linked to this PR (unintentionally) so idk if that is also contributing to the error.

**Pull request merged**